### ISSUE TITLE
No settings to default the cover as the first item of a collection

### ISSUE THREAD

**Issue by aureliengamet** [deckerst/aves#1180]
**Is your feature request related to a problem? Please describe.**

I use Aves to read mangas on my phone and my tablet, basically browsing folders with JPGs in them. It's a bit frustrating to me that the app auto select the last image of the folder as the cover, and doesn't have an option to pick the first one by default. For mangas, the first image is always the cover, which is what I want for my usecase. I manually click "Custom" to select the first image for each volume, but it's a bit cumbersome and time consuming.

**Describe the solution you'd like**

- A global setting to have Aves select the first item for all collections as the cover
- A third "First item" option in the cover picker, in addition to the already existing "Latest item" and "Custom"

**Describe alternatives you've considered**

Manually set the cover one by one using the "Custom" option of the cover picker.

**Additional context**
N/A

**Comment by deckerst** [comment]
I suppose you mean "first alphabetically" in your case, right? Maybe"first by date" would be needed too.

**Comment by aureliengamet** [comment]
> I suppose you mean "first alphabetically" in your case, right? Maybe"first by date" would be needed too.

Yes you are right, they're always sorted alphabetically in my usecase.
